modules/administration-guide/examples/proc_che-manually-upgrading-che-7-41-to-7-49-on-openshift.adoc

```diff
@@ -0,0 +1,77 @@
+:_content-type: PROCEDURE
+
+:parent-context-of-manually-upgrading-{prod-id-short}-{prod-last-version-pre-dwo}-to-7.49-on-openshift: {context}
+
+[id="manually-upgrading-{prod-id-short}-{prod-last-version-pre-dwo}-to-7.49-on-openshift_{context}"]
```

Comment:
Let's hardcode this, too. The ID should mirror the title.
```suggestion
[id="manually-upgrading-che-7.41-to-7.49-on-openshift_{context}"]
```

Reply:
No. IDs mirror titles. If downstream is different, fix it there, too, please.

Reply:
OK, will do.